cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
This `TicketSerializer` is a code quality issue. It's overriding the `validate` method to call a class method on the `Ticket` model. This is mixing model validation with serialization, which can lead to confusion. Consider moving the validation logic to the `Ticket` model's `clean` method, and call `full_clean` in the serializer's `create` or `update` methods before saving the object.